dao-services/src/main/java/org/exoplatform/wiki/jpa/JPADataStorage.java

```diff
@@ -59,6 +61,9 @@
 public class JPADataStorage implements DataStorage {
   public static final String WIKI_TYPE_DRAFT = "draft";
 
+  public static final String WIKI_FILES_NAMESPACE_NAME = "wiki-attachment";
```

Comment:
We had in mind to have 1 namespace per application. So we shold rename it to "wiki" here IMO.